droopler.install

```diff
@@ -1324,3 +1324,24 @@ function droopler_update_8112() {
   $updater->checklist()
     ->saveProgress(['v2.1' => $values]);
 }
+
+/**
+ * Droopler 2.1 update d_product configuration.
+ */
+function droopler_update_8113() {
+  $updater = \Drupal::service('d_update');
+
+  $configs = [
+    'd_product' => [
+      'image.style.product_main_image' => '',
```

Comment:
Zdaje się że to pójdzie tylko jeśli taki config jeszcze nie istnieje. Może się mylę, albo trzeba tu podać hash albo `override` żeby nadpisać config na siłę.

Reply:
Odpalałem hook update z UI i zmieniło poprawnie konfigurację, miałem nowe style, a  o to w sumie chodziło?

Reply:
Ciekawe, powinno pokazać że config już istnieje i go nie wciągać (bo jest podane `=> ''`).
Przy `=> '7sd976sd90678908f7s856587f9070sdf'` powinno sprawdzić czy config się zmienił w stosunku do hasha
Przy `=> 'override` powinno nadpisać config bezwarunkowo

Reply:
![image](https://user-images.githubusercontent.com/37533838/81289307-7d365c80-9066-11ea-96da-b6ce624bb2dd.png)
jak jest pusty to brutalnie nadpisuje :)

Reply:
w zasadzie to override jest takie jakby lekko zbędne tu... :thinking: 
chyba przypadkiem wyszedł taki drobny budżek ;)

Reply:
No właśnie nie powinno tak być, pusty hash oznaczał że config jest nowy - i jeśli okazałoby się inaczej to update by się nie wykonał. Override miało nadpisać go nie sprawdzając.